ar4_isaac/run_sim.py

```diff
@@ -106,18 +131,28 @@ def main(args):
 
     open_stage(simulation_app, args.stage_path)
 
+    world = create_world(
+        simulation_app, args.physics_dt, args.rendering_dt, args.stage_units_per_meter
+    )
+
     spawn_ar4(simulation_app, args.robot_model_path)
 
-    while True:
-        simulation_app.update()
+    rate = set_rate(simulation_app, args.sim_dt)
+
+    while simulation_app.is_running():
+        world.step()
+        rate.sleep()
```

Comment:
I don't think we need to set another rate, just calling step as fast as possible will make it right.

```suggestion
    while simulation_app.is_running():
        world.step()
```

[Example](https://github.com/isaac-sim/OmniIsaacGymEnvs/blob/f8f91bcf73cfd8ede2eab70e19dd70162abf9775/omniisaacgymenvs/utils/terrain_utils/create_terrain_demo.py#L170-L176)

If we do this, I think we are computing 4 physics steps on each world.step() call.

Reply:
Hmm nvm I guess if we do that we get a `RTF` of ~3 and the goal of this PR is to clamp it at 1.